lib/3scale_toolbox/commands/plans_command/import/step.rb

```diff
@@ -82,8 +98,14 @@ def find_feature_by_system_name(system_name)
             service_features.find { |feature| feature['system_name'] == system_name }
           end
 
-          def find_metric_by_system_name(system_name)
-            service_metrics_and_methods.find { |metric| metric.system_name == system_name }
+          def find_metric(metric_system_name, backend_system_name)
```

Comment:
The idea of this method is that only one of the two can be nil right?

Reply:
No, metric_system_name must always be non-nil. backend_system_name can be nil. Maybe we can have it with a default value to make it explicit.

```
          def find_metric(metric_system_name, backend_system_name = nil)
```